src/projections.jl

```diff
@@ -52,26 +53,41 @@ function matrix_projection(
     return result
 end
 
+function matrix_projection(
+    vals::AbstractVector{T},
+    ccls::AbstractVector{<:AbstractOrbit{<:AbstractMatrix}},
+    dim = size(first(first(ccls)), 1),
+) where T
+
+    return sum(val .* sum(g->inv(Matrix(g)), cc) for (val, cc) in zip(vals, ccls))
```

Comment:
For large matrices with few elements, it might be easier to compute the inverse by looking over all other elements of the group and finding which one is your inverse :-P

Reply:
Not it might, it almost always is. This is what we do in case `g::GroupElement`, but here `g` is already a sparse matrix (and you can't invert a sparse matrix).